e2e/tests/functional/plugins/notebook/notebook.e2e.spec.js

```diff
@@ -293,40 +295,51 @@ test.describe('Notebook entry tests', () => {
 
         expect(await validLink.count()).toBe(1);
     });
-    test.fixme('when an invalid link is entered into a notebook entry, it does not become clickable when viewing', async ({ page }) => {
+    test('when an invalid link is entered into a notebook entry, it does not become clickable when viewing', async ({ page }) => {
         const TEST_LINK = 'www.google.com';
-        await page.goto('./#/browse/mine', { waitUntil: 'networkidle' });
 
-        // Create Notebook
-        const notebook = await createDomainObjectWithDefaults(page, {
-            type: 'Notebook',
-            name: "Entry Link Test"
-        });
+        const showSelectedItemButton = page.getByTitle('Show selected item in tree');
 
-        await expandTreePaneItemByName(page, 'My Items');
+        // Navigate to the notebook object
+        await page.goto(notebookObject.url);
 
-        await page.goto(notebook.url);
+        // Reveal the notebook in the tree
+        await showSelectedItemButton.click();
 
         await nbUtils.enterTextEntry(page, `This should NOT be a link: ${TEST_LINK} is it?`);
 
         const invalidLink = page.locator(`a[href="${TEST_LINK}"]`);
 
         expect(await invalidLink.count()).toBe(0);
     });
-    test.fixme('when a nefarious link is entered into a notebook entry, it is sanitized when viewing', async ({ page }) => {
+    test('when a link is entered, but it is not in the whitelisted urls, it does not become clickable when viewing', async ({ page }) => {
+        const TEST_LINK = 'http://www.bing.com';
```

Comment:
i can dig this

Reply:
That was for you.

Reply:
let's update the variable definition to be INVALID_TEST_LINK

Reply:
got it!